PhysicsExploration/main.cpp

```diff
@@ -1,12 +1,17 @@
-#include "App.h"
+#include "ModelLoading.h"
+#include "PhysicsSimulationsFactory.h"
 using namespace std;
 
 int main()
-{
-	App* application = new App();
```

Comment:
Did you delete the App files? I don't see their deletion in the change list

Reply:
I though we have discussion before eliminating it :)

Reply:
You're free to do so!

Reply:
i saw you deleted App.h but not App.cpp...